workflows/tree_inference.smk

```diff
@@ -4,20 +4,28 @@ import json
 
 from pyggdrasil.tree_inference import McmcConfig, MoveProbConfig, MoveProbConfigOptions, McmcConfigOptions
 
+###############################################
+## Relative path from DATADIR to the repo root
+
+REPODIR = "/cluster/work/bewi/members/gkoehn/repos/PYggdrasil"
+
+###############################################
+
 
 rule make_random_or_deep_tree:
     """Make a tree (TreeNode) of random/deep topology and save it as JSON."""
     input:
-        script="../scripts/make_tree.py"
+        # TODO (Gordon): make this flexible - repo/PYggdrasil is not always the working directory
```

Comment:
Yes, good idea :slightly_smiling_face:  Perhaps it's worth adding a low-priority issue.